src/main/java/org/prebid/server/bidder/smaato/SmaatoBidder.java

```diff
@@ -162,37 +166,38 @@ private Site modifySite(Site site, String firstPublisherId) {
     }
 
     private User modifyUser(User user) {
-        final ExtUser ext = user.getExt();
-        if (ext == null) {
-            return user;
+        if (user == null) {
+            return null;
         }
 
-        final ObjectNode extDataNode = ext.getData();
-        if (extDataNode == null) {
+        final ExtUser userExt = user.getExt();
+        if (userExt == null || userExt.getData() == null || userExt.getData().isNull()) {
```

Comment:
Minor. Since `data` is ObjectNode we can do the following: 
```
final ObjectNode extDataNode = userExt != null ? userExt.getData() : null;
if (extDataNode == null) {
    return user;
}
```

Reply:
It is not critical and your solution even is more safe, so just fyi.